apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/CoverageTransformer.java

```diff
@@ -137,6 +137,99 @@ private static Coverage transformPartA(MetricRegistry metricRegistry, Beneficiar
               coverage, CcwCodebookVariable.A_TRM_CD, beneficiary.getPartATerminationCode()));
     }
 
+    // The reference year of the enrollment data
```

Comment:
Does any of this data need to be added to the reference files used in the unit tests, to ensure the assertions are still accurate?  
I noticed some failing unit tests in the Github Actions CI... could it be because the `rfrnc_yr` was not added to [these resource files](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses) used in unit test comparison?

Reply:
Thanks for the tip. Updated those resources to reflect the new coverage responses expected. Tests now pass.